sycl/include/CL/sycl.hpp

```diff
@@ -11,7 +11,7 @@
 #include <CL/sycl/accessor.hpp>
 #include <CL/sycl/atomic.hpp>
 #include <CL/sycl/backend.hpp>
-#include <CL/sycl/backend/opencl.hpp>
+//#include <CL/sycl/backend/opencl.hpp>
```

Comment:
Do we really need to include this always?

Reply:
I think, we don’t. The header includes some OpenCL stuff, which may be unavailable on user machine.

Tagging @romanovvlad @bader @jbrodman @rolandschulz